lib/sweet-alert.js

```diff
@@ -214,10 +231,9 @@
     switch (typeof arguments[0]) {
 
       case 'string':
-        params.title              = arguments[0];
-        params.text               = arguments[1] || '';
-        params.type               = arguments[2] || '';
-        params.confirmButtonClass = params.type ? 'btn-' + params.type : defaultParams.confirmButtonClass;
```

Comment:
That was a mistake. I am sorry, there is no need to delete that. What i did was copied the fix from the original sweet alert repository. And they didnt had it back then.